presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -2136,6 +2136,91 @@ public void testCommentTable()
+                        "   format = 'TEXTFILE',\n" +
+                        "   textfile_skip_header_line_count = 1\n" +
+                        ")",
+                getSession().getCatalog().get(),
+                getSession().getSchema().get());
+
+        assertUpdate(createTableSql);
+
+        MaterializedResult actual = computeActual("SHOW CREATE TABLE test_table_skip_header");
+        assertEquals(actual.getOnlyValue(), createTableSql);
+
+        assertUpdate("INSERT INTO test_table_skip_header VALUES ('header'), ('value'), ('footer')", 3);
+        assertQuery("SELECT * FROM test_table_skip_header", "VALUES ('value'), ('footer')");
+        assertUpdate("DROP TABLE test_table_skip_header");
+
+        createTableSql = format("" +
+                        "CREATE TABLE %s.%s.test_table_skip_footer (\n" +
+                        "   name varchar\n" +
+                        ")\n" +
+                        "WITH (\n" +
+                        "   format = 'TEXTFILE',\n" +
+                        "   textfile_skip_footer_line_count = 1\n" +
+                        ")",
+                getSession().getCatalog().get(),
+                getSession().getSchema().get());
+
+        assertUpdate(createTableSql);
+
+        actual = computeActual("SHOW CREATE TABLE test_table_skip_footer");
+        assertEquals(actual.getOnlyValue(), createTableSql);
+
+        assertUpdate("INSERT INTO test_table_skip_footer VALUES ('header'), ('value'), ('footer')", 3);
+        assertQuery("SELECT * FROM test_table_skip_footer", "VALUES ('header'), ('value')");
+        assertUpdate("DROP TABLE test_table_skip_footer");
+
+        createTableSql = format("" +
+                        "CREATE TABLE %s.%s.test_table_skip_header_footer (\n" +
+                        "   name varchar\n" +
+                        ")\n" +
+                        "WITH (\n" +
+                        "   format = 'TEXTFILE',\n" +
+                        "   textfile_skip_footer_line_count = 1,\n" +
+                        "   textfile_skip_header_line_count = 1\n" +
+                        ")",
+                getSession().getCatalog().get(),
+                getSession().getSchema().get());
+
+        assertUpdate(createTableSql);
+
+        actual = computeActual("SHOW CREATE TABLE test_table_skip_header_footer");
+        assertEquals(actual.getOnlyValue(), createTableSql);
+
+        assertUpdate("INSERT INTO test_table_skip_header_footer VALUES ('header'), ('value'), ('footer')", 3);
+        assertQuery("SELECT * FROM test_table_skip_header_footer", "VALUES ('value')");
+        assertUpdate("DROP TABLE test_table_skip_header_footer");
+    }
+
+    @Test
+    public void testCreateTableInvalidSkipHeaderFooter()
+    {
+        assertThatThrownBy(() -> assertUpdate("CREATE TABLE test_orc_skip_header (col1 bigint) WITH (format = 'ORC', textfile_skip_header_line_count = 1)"))
+                .isInstanceOf(RuntimeException.class)
```

Comment:
Nit: asserting `RuntimeException` here isn't useful. Since it's not a specific exception type, all we care is that the message matches.